doc/source/parameters/dem/lagrangian_physical_properties.rst

```diff
@@ -7,14 +7,8 @@ In this subsection, gravitational acceleration, and the physical properties of t
 .. code-block:: text
 
   subsection lagrangian physical properties
-    # Gravitational acceleration in x direction
-    set gx                       = 0.0
-
-    # Gravitational acceleration in y direction
-    set gy                       = 0.0
-
-    # Gravitational acceleration in z direction
-    set gz                       = -9.81
+    # Gravitational acceleration vector
+    set g                       = 0.0, 0.0, 0.0
```

Comment:
Did you change the default value?

Reply:
No it was always set to 0.0 for all directions in the code, the doc was wrong